ui/.env.production

```diff
@@ -1,8 +1,8 @@
 INLINE_RUNTIME_CHUNK=false
-PUBLIC_URL=__PUBLIC_PATH_PREFIX__/
+PUBLIC_URL=__PUBLIC_PATH_PREFIX__
 REACT_APP_VERSION=$npm_package_version
-REACT_APP_NAME=$npm_package_name
 REACT_APP_SENTRY_DSN=https://10930d3b0a8d427cad2147d6d845be56@o226447.ingest.sentry.io/5721090
 REACT_APP_SENTRY_ENABLED=true
 # REACT_APP_RELEASE_VERSION is set in config-overrides.js
 REACT_APP_RELEASE_VERSION=unknown
+REACT_APP_DASHBOARD_API_URL=''
```

Comment:
We need set all env variables' default value, right?

Reply:
Nope, I found there is a way to work around it:

```js
// process-slim.js
export let process = {
  cwd: () => '',
  env: {}
}
```

I test and it works fine.